src/main/java/com/thebois/persistence/AbstractPersistence.java

```diff
@@ -0,0 +1,33 @@
+package com.thebois.persistence;
+
+import java.io.File;
+
+/**
+ * Basic foundation for persistence, common functionality for all persistence classes.
+ */
+abstract class AbstractPersistence {
+
+    private final String directory =
+        System.getProperty("user.home") + "/Documents/Pawntastic/saves/";
+
+    /**
+     * All subclasses need to use this in the constructor to ensure the directory is there before
+     * any commands/operations can be carried out.
+     */
+    protected void directoryExist() {
+        final File filePath = new File(directory);
+        if (!filePath.exists()) {
+            filePath.mkdirs();
+        }
+    }
+
+    /**
+     * Subclasses uses the same save-file.
```

Comment:
This method doesn't really have anything to do with subclasses, more than the fact that it is `protected`, which allows them to call it. It's imo a bit unnecessary to mention subclasses in the javadoc. Just "Gets the path to the save file." is enough.